### DISCUSSION TITLE
Can I load specific plugin *after* loading `lua/config/` files?

### DISCUSSION THREAD

**Discussion by boltlessengineer** [LazyVim/LazyVim#268] [Q&A] [answered]
I'm using plugin [auto-hlsearch.nvim](https://github.com/asiryk/auto-hlsearch.nvim), which overwrites my user-defined keymaps. So I have to map my keymaps first, and then run `require('auto-hlsearch').setup({...})`.

But LazyVim loads plugin before `lua/config/keymap.lua`, so this plugin is breaking because `lua/config/keymap.lua` overwrites it (should be reverse).

Are there any autocmd-event executes after loading `lua/config/` files?

I tried `event = 'CursorMoved'`. But this didn't solve the problem.

**Comment by folke** [comment, marked as answer]
Try loading hlsearch on `VeryLazy`

**Reply by boltlessengineer** [reply]
It still doesn't work. `lua/config/keymap.lua` overwrites auto-hlsearch.

When I don't map `n`/`N` keys (which should be overwritten by auto-hlsearch) in `keymaps.lua` file, auto-hlsearch works fine.

**Reply by folke** [reply]
Change the `config` function to somthing like `vim.schedule(function() require('auto-hlsearch').setup({...}) end)`

This way it will run for sure after the keymaps are loaded

**Reply by boltlessengineer** [reply]
That works. Thanks!!
